app/controllers/placements/providers_controller.rb

```diff
@@ -1,5 +1,11 @@
 class Placements::ProvidersController < Placements::ApplicationController
   def show
-    @provider = current_user.providers.find(params.require(:id)).decorate
+    @provider = providers.find(params.require(:id)).decorate
+  end
+
+  private
+
+  def providers
+    current_user.support_user? ? Placements::Provider.all : current_user.providers
   end
```

Comment:
Let's definitely lean in to policy scopes for this stuff! I have a feeling you've already planned to do this in a separate PR?

Reply:
Oh! Is it #1101?? 👀